### PR TITLE
fix(StorageNodes): make fqdn similar to nodes page

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: src/utils/tooltip.js [hunk 1]
```diff
@@ -109,34 +111,6 @@ const NodeTooltip = (props) => {
     );
 };
 
-const NodeEndpointsTooltip = (props) => {
```
Comment (artemmufazalov) [inline src/utils/tooltip.js:112]: I don’t like two things in these tooltips:
1. redux reducer as additional level - it complicates code
2. two different ways to display tooltips in the app - this redux tooltip and different components from uikit

So I prefer to use Popover, Popup and Tooltip from uikit, it makes it easier and more consistent. Therefore I move this components from here to /components/Tooltips

Comment (Raubzeug) [reply src/utils/tooltip.js:112]: I totally agree with you! Redux Tooltip is awful! Let's get rid of it step by step.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/Storage/StorageNodes/StorageNodes.scss [hunk 2]
```diff
@@ -24,13 +24,10 @@
...
-        display: inline-block;
-        overflow: hidden;
-
-        max-width: 330px;
+        display: flex;
+        align-items: center;
 
-        vertical-align: top;
-        text-overflow: ellipsis;
+        max-width: 305px;
```
[the remark this answers is not in the document]
Comment (artemmufazalov) [reply src/containers/Storage/StorageNodes/StorageNodes.scss:30]: In this case overflow behaviour is defined in `EntityStatus` component, so we don't need to define it additionally

File: src/containers/Nodes/getNodesColumns.tsx [hunk 3]
```diff
@@ -47,27 +48,29 @@ export function getNodesColumns({
...
-                                className={b('external-button')}
-                                target="_blank"
-                            >
-                                <Icon name="external" />
-                            </Button>
-                        )}
+                        <Popover
+                            content={<NodeEndpointsTooltip data={row} />}
+                            placement={['top', 'bottom']}
+                            behavior={PopoverBehavior.Immediate}
```
[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/containers/Nodes/getNodesColumns.tsx:54]: Ok!

File: src/components/Tooltips/NodeEndpointsTooltip/index.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export * from './NodeEndpointsTooltip';
```
[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/components/Tooltips/NodeEndpointsTooltip/index.ts:1]: Commonly we don't use such technique in other projects. It's useful for libs when you collect imports from a few files. But here we save a few letters when import a component and have a pile of `index` files.

File: src/components/Icon/Icon.tsx [hunk 5]
```diff
@@ -31,3 +31,9 @@ export const Icon = ({
         />
     );
 };
+
+// When used with uikit components Icon is considered as text element and corresponding styles are applied
+// IconWrapper overrides displayName to 'Icon', so it will be considered as an icon with right styles
+export const IconWrapper = (props: IconProps) => <Icon {...props} />;
+
+IconWrapper.displayName = 'Icon';
```
Comment (Raubzeug) [inline src/components/Icon/Icon.tsx:39]: I realised that we even do not need Wrapper.
We may just do `Icon.displayName = 'Icon'` and that's it!

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/components/Icon/Icon.tsx (hunk 5 above)
Comment (Raubzeug) [reply src/components/Icon/Icon.tsx:39]: Ok, lets do an issue and refactor it in the future!

**Review by Raubzeug — APPROVED** [review]

File: src/components/Icon/Icon.tsx (hunk 5 above)
Comment (artemmufazalov) [reply src/components/Icon/Icon.tsx:39]: Added issue - https://github.com/ydb-platform/ydb-embedded-ui/issues/344

**Pull request merged**